dvc/command/metrics.py

```diff
@@ -245,7 +244,7 @@ def add_parser(subparsers, parent_parser):
     metrics_add_parser.add_argument("path", help="Path to a metric file.")
     metrics_add_parser.set_defaults(func=CmdMetricsAdd)
 
-    METRICS_MODIFY_HELP = "Modify metric file options."
+    METRICS_MODIFY_HELP = "Modify metric file values."
```

Comment:
it should be options.

`modifying values` sounds like we actually modify metric values.

it is about setting up metric file settings (type, path, etc), not modifying metric values

Reply:
I see. I'll update this... ⏳  but still need the PR for other strings.

Reply:
What about "default formatting"? See latest version here and in iterative/dvc.org/pull/1014

Reply:
`default formatting options`  is still better? )

Reply:
I'm trying to avoid "options" here since there are also command options and config options and it gets confusing some times in the explanations.

Reply:
Yep, I got the idea. I think it's almost impossible to avoid unfortunately in general (even if it works in this case). At least I was not able to find a good solution - may be even primarily I never thought that it's a really big deal - usually it's clear from the context. And can add more words when it's not clear from the context I think.

Reply:
I think I managed to avoid the term everywhere actually 🙂 assuming we let this one through. Yes it's not always a big deal but in this case and others I changed I thought it was confusing since all the "option" concepts are used in the same doc or even section of the doc sometimes.